src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/LinkedCasesMappingTest.java

```diff
@@ -1,58 +1,91 @@
 package uk.gov.hmcts.reform.sscs.helper.mapping;
 
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
 import uk.gov.hmcts.reform.sscs.ccd.domain.*;
+import uk.gov.hmcts.reform.sscs.ccd.service.CcdService;
+import uk.gov.hmcts.reform.sscs.exception.GetCaseException;
+import uk.gov.hmcts.reform.sscs.idam.IdamService;
 import uk.gov.hmcts.reform.sscs.model.service.linkedcases.ServiceLinkedCases;
+import uk.gov.hmcts.reform.sscs.service.CcdCaseService;
 
 import java.util.ArrayList;
 import java.util.List;
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.when;
+import static org.mockito.MockitoAnnotations.openMocks;
 
+@ExtendWith(MockitoExtension.class)
 class LinkedCasesMappingTest {
+    private CcdCaseService ccdCaseService;
 
-    public static final long CASE_ID = 99250807409918L;
+    @Mock
+    private CcdService ccdService;
+
+    @Mock
+    IdamService idamService;
+
+    @BeforeEach
+    void setUp() {
+        openMocks(this);
```

Comment:
I'd use

`
@InjectMocks
CcdCaseService ccdCaseService
`
At class level instead of openmocks